shared/settings/advanced/index.js

```diff
@@ -105,9 +118,7 @@ class Developer extends React.Component<Props, State> {
     return (
       <Kb.Box style={styles.developerContainer}>
         <Kb.Text center={true} type="BodySmallSemibold" onClick={this._onLabelClick} style={styles.text}>
-          {Styles.isMobile
-            ? `Please don't do anything here unless instructed to by a developer.`
```

Comment:
surely people don't need developers to tell them to enable 'Forbid account changes from the website'?

Reply:
thats not what the intention was for that setting. it was for settings which aren't for the regular user (aka db nuke)

Reply:
Right, but "Please don't do anything here" sounds like it includes 'Forbid account changes from the website'. Desktop has 'Please don't do anything below' which also seems to be clearer for mobile.